ISHPullUp.xcodeproj/project.pbxproj

```diff
@@ -280,9 +291,12 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
-				INFOPLIST_FILE = ISHPullUp/Info.plist;
+				INFOPLIST_FILE = "$(SRCROOT)/Supporting Files/Info.plist";
```

Comment:
Given that this folder is also used in URLs, we should consider removing the space.